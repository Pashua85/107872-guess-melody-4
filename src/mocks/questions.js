const AVATAR_URL = `https://api.adorable.io/avatars/17`;

export default [
  {
    type: `artist`,
    questionText: `Кто исполняет эту песню?`,
    song: {
      artist: `Gregory Porter`,
      src: `someSrc-4`
    },
    answers: [{
      picture: `${AVATAR_URL}/${Math.random()}`,
      artist: `Kurt Elling`
    }, {
      picture: `${AVATAR_URL}/${Math.random()}`,
      artist: `Gregory Porter`
    }, {
      picture: `${AVATAR_URL}/${Math.random()}`,
      artist: `Tony Bennett`
    }]
  },
  {
    type: `genre`,
    questionText: `Выберите музыку барокко`,
    genre: `baroque`,
    answers: [{
      src: `someSrc-0`,
      genre: `baroque`
    }, {
      src: `someSrc-1`,
      genre: `classical`
    }, {
      src: `someSrc-2`,
      genre: `romantic`
    }, {
      src: `someSrc-3`,
      genre: `minimalism`
    }]
  }
];
