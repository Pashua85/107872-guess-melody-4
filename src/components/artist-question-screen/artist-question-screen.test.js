import React from 'react';
import {MemoryRouter} from 'react-router-dom';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import ArtistQuestionScreen from './artist-question-screen';
import questions from '../../mocks/test-questions';

Enzyme.configure({
  adapter: new Adapter()
});

describe(`ArtistQuestionScreen`, () => {
  it(`should render correctly`, () => {
    const wrapper = shallow(
        <MemoryRouter>
          <ArtistQuestionScreen
            question={questions[0]}
            onAnswerSelect={() => {}}
          />
        </MemoryRouter>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
