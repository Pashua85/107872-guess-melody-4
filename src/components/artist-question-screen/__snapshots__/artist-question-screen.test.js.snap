// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArtistQuestionScreen should render correctly 1`] = `ShallowWrapper {}`;
